nengo_deeplearning/utils.py

```diff
@@ -292,8 +292,11 @@ def minibatch_generator(inputs, targets, minibatch_size, shuffle=True,
     rng : :class:`~numpy:numpy.random.RandomState`, optional
         random number generator
 
-    Yields
-    ------
+    Returns
+    -------
+
+    A generator that yeilds:
```

Comment:
Apparently it was added in 0.6 https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt